src/TinyIoC/TinyIoC.cs

```diff
@@ -3173,7 +3182,11 @@ private void AutoRegisterInternal(IEnumerable<Assembly> assemblies, DuplicateImp
                     {
                         RegisterInternal(type, string.Empty, GetDefaultObjectFactory(type, type));
                     }
+#if PORTABLE
+                    catch (MemberAccessException)
+#else
                     catch (MethodAccessException)
+#endif

```

Comment:
I'm not sure about this change.. Previously it was caching MemberAccessException which was working in pcl... But the latest update broke it as MethodAccessException doesn't exist in pcl. So I'm catching these differently... I think it might be nice to just catch all exceptions and write it out to the debug log and rethrow if it's a tiny exception... Thoughts?